docs/data/material/components/alert/alert.md

```diff
@@ -8,83 +8,127 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/alert/
 
 # Alert
 
-<p class="description">An alert displays a short, important message in a way that attracts the user's attention without interrupting the user's task.</p>
+<p class="description">Alerts display brief messages for the user without interrupting their use of the app.</p>
 
-:::info
-This component is not documented in the [Material Design guidelines](https://m2.material.io/), but it is available in Material UI.
+## Introduction
+
+The Alert component can be used to provide important and potentially time-sensitive information in a way that does not interfere with the user's tasks. (Source: [ARIA APG](https://www.w3.org/WAI/ARIA/apg/patterns/alert/).)
+
+This component is not documented in the [Material Design guidelines](https://material.io/).
+
+:::success
+Alerts should not be confused with alert _dialogs_ ([ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/)), which _are_ intended to interrupt the user to obtain a response.
+Use the Material UI [Dialog](https://mui.com/material-ui/react-dialog/) if you need this behavior.
 :::
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
```

Comment:
I feel like the best place to position this component here is right below the description (or, effectively, below the ad container). What do you think? It's how we did in the recent Divider and Accordion pages. It looks like a cohesive set of very introductory links — particularly on the Base UI docs, where the ad container is below the tabs (a layout we should get to, eventually!).